src/hazelweb/gui/ActionPanel.re

```diff
@@ -0,0 +1,464 @@
+    | n => n
+    };
+  };
+
+  KeyCombo.[
+    section(
+      "Movement",
+      [
+        info([text("Move using arrow keys")]),
+        dual_line("Move to next hole ", Tab, "Previous hole ", ShiftTab),
+      ],
+    ),
+    section(
+      "General Editing",
+      [
+        spaced_line([
+          text("Backspace / Delete"),
+          Vdom.Node.div(
+            [display_inline_block],
+            [
+              keyboard_button(Backspace) |> add_style(bottom_border_none),
+              keyboard_button(Delete) |> add_style(bottom_border_none),
+            ],
+          ),
+        ]),
+        combo(Ctrl_Alt_I, simple("Swap expression up")),
+        combo(Ctrl_Alt_K, simple("Swap expression down")),
+        combo(Ctrl_Alt_J, simple("Swap expression left")),
+        combo(Ctrl_Alt_L, simple("Swap expression right")),
+        combo(Enter, simple("Create new line ")),
+        combo(LeftParen, simple("Parenthesize")),
+      ],
+    ),
+    section(
+      "Arithmetic",
+      [
+        info([
+          text("Enter number literals directly e.g. "),
+          mono_text("1.0, 2"),
+        ]),
+        info_action(
+          [
+            text("Type "),
+            mono_text("\"I\""),
+            text(" to insert an Integer type"),
+          ],
+          Action.Construct(SChar("I")),
+        ),
+        info_action(
+          [
+            text("Type "),
+            mono_text("\"F\""),
+            text(" to insert a Float type"),
+          ],
+          Action.Construct(SChar("F")),
+        ),
+        spaced_line([
+          text("Floating point operators, type"),
+          Vdom.Node.div(
+            [],
+            [
+              brown_label([mono_text("+.")]) |> add_style(bottom_border_none),
+              brown_label([mono_text("-.")])
+              |> add_style(bottom_border_none),
+              brown_label([mono_text("*.")])
+              |> add_style(bottom_border_none),
+            ],
+          ),
+        ]),
+        operator_list(
+          ~on_type=false,
+          "Integer operators",
+          [Plus, Minus, Asterisk],
+        ),
+      ],
+    ),
+    section(
+      "Variables",
+      [
+        info([text("Variable regex: ^[_a-z][_a-zA-Z0-9']*$")]),
+        info_action(
+          [
+            text("Type "),
+            mono_text("\"let \""),
+            text(" to enter a let expression"),
+          ],
+          Action.Construct(SLet),
+        ),
+      ],
+    ),
+    section(
+      "Booleans",
+      [
+        info([
+          text("Enter boolean literals "),
+          mono_text("\"true\", \"false\""),
+          text(" directly"),
+        ]),
+        info_action(
+          [
+            text("Type "),
+            mono_text("\"B\""),
+            text(" to insert a Bool type"),
+          ],
+          Action.Construct(SChar("B")),
+        ),
+        operator_list(
+          ~on_type=false,
+          "Operators",
```

Comment:
these are actually integer comparison operators only. there are also dotted versions of these for float comparison. let's include these in the arithmetic section, and move the boolean section above it so that booleans are described before the comparison operators.